mars/dataframe/utils.py

```diff
@@ -12,33 +12,48 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import numpy as np
-import pandas as pd
-from mars.lib.mmh3 import hash as mmh_hash
-
 import itertools
 import operator
 import functools
 
+import numpy as np
+import pandas as pd
+from mars.lib.mmh3 import hash as mmh_hash
```

Comment:
Could be from `..lib.mmh3 import hash as mmh_hash` using a relative import, the old one must be wrong and not been discovered before.